projects/github.com/bazelbuild/bazelisk/package.yml

```diff
@@ -0,0 +1,32 @@
+distributable:
+  url: https://github.com/bazelbuild/bazelisk/archive/refs/tags/v{{version}}.tar.gz
+  ref: v{{version}}}
```

Comment:
`ref:` is used with git checkouts; you want `strip-compents: 1`, so the name-version of the package is stripped from the extraction path.

Reply:
done!